src/main.rs

```diff
@@ -225,23 +240,29 @@ impl Fixer {
             },
         };
 
-        if let Some(func_info) = file_info.func_info(address) {
-            let name = func_info.demangled_name();
-            if let Some(line_info) = func_info.line_info(address) {
+        let (mut fn_name, mut location) = if let Some(func_info) = file_info.func_info(address) {
```

Comment:
Double assignments on one line can be confusing, please split the logic that finds the function name and location to a separate function.

Reply:
I have changed it a bit so the double assignment isn't necessary.